### DISCUSSION TITLE
Sun2000 -6ktl - FW136

### DISCUSSION THREAD

**Discussion by Criaga** [wlcrs/huawei_solar#609] [Connecting to the inverter]
Hello, I saw that there is an update on the FusionSolar website, version 136 for the inverter.
 Have you updated ? 
Are there any problems with integration? 
Thank you

**Comment by miguelcrego** [comment]
I upgrade and no problem

**Comment by JFHCFC** [comment]
Huawei Solar 1.3.1 working with:

SUN2000-10KTL-M1 - V100R001C00SPC161
SDongleA-05 - V100R001C00SPC133
LUNA2000 - V100R002C00SPC127
SUN2000-450W-P2 - V100R002C10SPC019

working well
